tmt/steps/discover/shell.py

```diff
@@ -336,9 +337,13 @@ def go(self) -> None:
         # (unless remote repository is provided using 'url')
         if not url:
             assert self.step.plan.worktree  # narrow type
+
             relative_path = os.path.relpath(self.step.plan.worktree, self.workdir)
             os.symlink(relative_path, testdir)
 
+            # relative_path = self.step.plan.worktree.relative_to(self.workdir)
```

Comment:
Perhaps include a short `TODO` comment?

Reply:
Not needed, actually, I removed those two lines in c91be698.

Reply:
Good.